dotnet/xml/ns-Microsoft.ML.SamplesUtils.xml

```diff
@@ -1,8 +1,10 @@
 <Namespace Name="Microsoft.ML.SamplesUtils">
   <Docs>
     <summary>
-      Namespace containing utilities to download and featurize datasets for the API reference samples.
-      It is part of the Microsoft.ML.SampleUtils nuget package.
+    Namespace containing utilities to download and featurize datasets for the API reference samples.
     </summary>
+    <remarks>
+    Included in the Microsoft.ML.SampleUtils nuget package.
```

Comment:
>Microsoft.ML.SampleUtils [](start = 20, length = 24)

link to: [https://www.nuget.org/packages/Microsoft.ML.SampleUtils](https://www.nuget.org/packages/Microsoft.ML.SampleUtils)